js/modules/bookList.js

```diff
@@ -0,0 +1,80 @@
+export default class BookListing {
+  // declare a constructor function
+  constructor() {
+    if (localStorage.getItem('bookList')) {
+      this.list = JSON.parse(localStorage.getItem('bookList'));
+    } else {
+      this.list = [];
+    }
+  }
+
+  // create a display method
+  displayList() {
```

Comment:
[ Required ] As requested by a previous reviewer please make sure that the following methods in class `BookListing` are written using  `ES6` arrow functions syntax. The two methods are;
- displayList()
- addToList()
- removeBookFromList(index)

You can achieve this by using the code below

```JavaScript
  class BookListing {
     displayList = () => {
       // Body of the method
     }

    addToList = (newBook) => {
      // Body of the method
     }
     
     removeBookFromList = (index) => {
        // Body of the method
    }
  }
```

Reply:
You can't use arrow function with class methods, see MDN documentation below:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions

![Screenshot_20221108-224650_Chrome](https://user-images.githubusercontent.com/22054464/200683324-13255feb-72c9-4752-b91f-818178fef121.jpg)

Reply:
Hi @fmanimashaun, You can use arrow functions in classes without the keyword `const`. Please try writing the method as I have illustrated below and it will work.

Reply:
Sorry! 

> Hi @fmanimashaun, You can use arrow functions in classes without the keyword `const`. Please try writing the method as I have illustrated below and it will work.
> 
> ```js
>     class BookListing {
>      displayList = () => {
>        // Body of the method
>      }
> 
>     addToList = (newBook) => {
>       // Body of the method
>      }
>      
>      removeBookFromList = (index) => {
>         // Body of the method
>     }
>   }
> ```

There is nowhere in the documentation that says one must use the arrow function for methods in classes and I didn't use the `const` keyword in my method declaration. below is a typical implementation of a method on my code:
```
  addToList(newBook) {
    // update the book list with the new book
    this.list.push(newBook);

    // update the local storage with the new books
    localStorage.setItem('bookList', JSON.stringify(this.list));

    // update the page with the new book
    this.displayList();
  }
```

which is consistent with the documentation on ES6 as shown below:

![reference](https://user-images.githubusercontent.com/22054464/200763384-2063928e-a9f5-4486-8e83-642d0fe064e1.png)

Reply:
Sorry, I didn't say it's a must. You indicated in your statement above that it's impossible to use arrow functions in classes. I was just making a point that it's possible, the goal of this exercise was to use `ES6` syntax in your `Awesome Books` project that's why the first reviewer and I requested the use of arrow functions in your `BookListing` class.